### PR TITLE
CH-549

### PR DESCRIPTION
**Pull request by michalbella**

### CODE REVIEW

File: src/components/connectButton/ConnectButton.tsx [hunk 1]
```diff
@@ -143,7 +144,8 @@ const ConnectButton = () => {
 											<SC.WalletRow justify={'space-between'}>
 												{/* empty alt text for decorative img */}
 												<Col flex={'auto'} style={{ padding: '0px 0px 0px 8px' }}>
-													{chain?.hasIcon && <SC.Logo alt={''} src={chain?.iconUrl} />}
+													{/* // TODO: if next network will be added need to be conditionaly rendered icon */}
```
Comment (daniel-hlavaty) [inline src/components/connectButton/ConnectButton.tsx:147]: Wasnt the optimism icon we used before... more sexy?

File: src/utils/constants.ts [hunk 2]
```diff
@@ -231,7 +231,8 @@ export enum SportFilterEnum {
 	Tennis = 'Tennis',
 	eSports = 'eSports',
 	Cricket = 'Cricket',
-	Motosport = 'Motosport',
+	// NOTE: Motosport is currently not available
```
Comment (daniel-hlavaty) [inline src/utils/constants.ts:234]: doesnt this only remove the option from filter? Why is it unavailable, should its matches even be shown in list?

**Review by daniel-hlavaty — CHANGES_REQUESTED** [review]

File: src/utils/constants.ts (hunk 2 above)
Comment (michalbella) [reply src/utils/constants.ts:234]: I just copied. this logic from Thales and yes I just dont want this sport in Filter until sezon starts.

File: src/components/connectButton/ConnectButton.tsx (hunk 1 above)
Comment (michalbella) [reply src/components/connectButton/ConnectButton.tsx:147]: yeah this is more red and sexy :)

**Pull request merged**